documentation/docs/json/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: JSON
+description: JSON parser and transformation
+index: 0
+module: json
+---
+
+Module: [![Maven Central](https://img.shields.io/maven-central/v/org.gnieh/fs2-data-json_2.13.svg)](https://mvnrepository.com/artifact/org.gnieh/fs2-data-json_2.13)
+
+The `fs2-data-json` module provides tools to parse, query and transform JSON data in a streaming manner.
+
+This page covers the following topics:
+* Contents
+{:toc}
+
+### Basic usage
+
+To create a stream of JSON tokens from an input stream, use the `tokens` pipe in `fs2.data.json` package. This pipe accepts a stream of characters and returns a stream of JSON tokens. This produces a stream of structurally valid tokens forming the JSON documents.
+
+```scala mdoc
+import cats.effect._
+
+import fs2._
+import fs2.data.json._
+
+val input = """{
+              |  "field1": 0,
+              |  "field2": "test",
+              |  "field3": [1, 2, 3]
+              |}
+              |{
+              |  "field1": 2,
+              |  "field3": []
+              |}""".stripMargin
+
+val stream = Stream.emits(input).through(tokens[IO])
+stream.compile.toList.unsafeRunSync()
+```
+
+The pipe validates the JSON structure while parsing. It reads all the json values in the input stream and emits tokens as they are available.
+
+### Selectors
+
+Selectors can be used to select a subset of a JSON token stream.
+
+For instance, to select and enumerate elements that are in the `field3` array, you can create this selector. Only the tokens describing the values in `field3` will be emitted as a result.
```

Comment:
I would mention and link the `interpolators` module here, people might not find it otherwise – who reads full docs ;)

Reply:
Right, I added a paragraph.